charts/argocd-apps/values-test.yaml

```
@@ -295,3 +295,18 @@ applications:
         value: secret  # TODO: set SECRET_KEY_BASE and make it a secret
       - name: GOVUK_APP_NAME
         value: collections
+- name: government-frontend
+  helmValues:
+    appImage:
+      repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/government-frontend
+      tag: latest  # To be edited by automation (not yet implemented).
+    replicaCount: 1
+    extraEnv:
+      - name: SECRET_KEY_BASE
+        value: secret  # TODO: set SECRET_KEY_BASE and make it a secret
```

I've set up an external secret for SECRET_KEY_BASE now so you can fix this TODO like so:
```suggestion
      - name: SECRET_KEY_BASE
        valueFrom:
          secretKeyRef:
            name: rails-secret-key-base
            key: SECRET_KEY_BASE
```

---

Thanks Chris, will take a look at this now.